🐛 [Bug report]: `npm run build` not working

### Descrcibe the bug?

site not building in local development & prod 

### Steps to reproduce the bug?

1. `npm i` to install dependencies
2. `npm run dev` to run the site 
3. when you want to build the site using `npm run build` it gives us the error 

### Expected behavior

.

### Desktop (Please provide your system information)

- Kali Linux

### Mobile (Please provide your device information)

_No response_

### Screenshot / Screenshare

![reactplay](https://github.com/reactplay/react-play/assets/88102392/e4f73ab3-d6e4-4f52-8b5a-b3ec4dc214d2)


### Relevant log output

_No response_

### Record

- [X] I have checked the existing [issues](https://github.com/reactplay/react-play/issues)
- [X] I have read the [Contributing Guidelines](https://github.com/reactplay/react-play/blob/main/CONTRIBUTING.md)
- [ ] I want to work on this issue

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

> > Hi friend. Just to know, why do you use "npm run build" and you have a problem with pnpm ? Any idea?
> 
> If we run `pnpm run build`, we still get an error

@priyankarpal can you post the error you're getting while building the project.


I have setup the project in gitpod and getting this error while running `npm run build`
```bash
Creating an optimized production build...
Failed to compile.

Module not found: Error: Can't resolve '../node_modules/open' in '/workspace/react-play/node_modules/plays/test'
```

yes same error @bhansa 

here is the error in vs code (local setup)
```bash
Creating an optimized production build...
Failed to compile.

Module not found: Error: Can't resolve '../node_modules/open' in '/home/pewpew/Desktop/react-play/node_modules/.pnpm/plays@0.7.0/node_modules/plays/test'
```

--- a/package.json
+++ b/package.json
@@ -38,7 +38,6 @@
     "lodash.memoize": "^4.1.2",
     "mathjs": "^11.11.2",
     "p5": "^1.7.0",
-    "plays": "^0.7.0",
     "react": "^18.2.0",
     "react-codemirror2": "^7.3.0",
     "react-color": "^2.19.3",

